api/api-service.js

```diff
@@ -9,20 +9,19 @@ var Seneca = require('seneca')
 
 var server = new Hapi.Server()
 
-server.connection({ 
+server.connection({
   port: PORT
 })
 
 server.register({
-  register: Chairo, 
+  register: Chairo,
   options:{
     seneca: Seneca({
       tag: 'api',
       log: 'silent',
-      legacy: { logging: false },
-      internal: { logger: require('seneca-demo-logger') },
       debug: {short_logs:true}
     })
+    .use('demo-logger')
```

Comment:
Done. I can confirm the logging goes through the custom logger from the start.
However, demo-logger won't actually show anything because it only shows logs with kind == "act"